contrib/opencensus-ext-django/setup.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2019, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from setuptools import find_packages
+from setuptools import setup
+from version import __version__
+
+setup(
+    name='opencensus-ext-django',
+    version=__version__,  # noqa
+    author='OpenCensus Authors',
+    author_email='census-developers@googlegroups.com',
+    classifiers=[
+        'Intended Audience :: Developers',
+        'Development Status :: 3 - Alpha',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: Apache Software License',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.4',
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+    ],
+    description='OpenCensus Django Integration',
+    include_package_data=True,
+    long_description=open('README.rst').read(),
+    install_requires=[
+        'Django >= 1.11.0, <= 1.11.20',
```

Comment:
Not `< 2.2.0`? We support versions later than `1.11.20` now.

Thinking about this again, setting a maximum version for ext dependencies might come back to bite us because either (1) we rely on dependencies following semantic versioning and set the max version as the next minor version to be released, in which case we fail if a patch update breaks our integration or (2) we set the max version as the current version (including the patch level) and are forced to update this file with each dependency release.

I don't have a proposed solution here, just noting that this might cause problems for dependencies that don't follow semvar or have different versioning semantics than we expect.

Reply:
I actually tried Django 2.x and it failed the Py3.6 system test. https://github.com/census-instrumentation/opencensus-python/commit/eae1dee960080e48f77f6b6a3def2926fc4f9b2f

I will try to put the maximum Django version that we support here.

Reply:
Ok, no need to fix it this PR. Can you open a ticket for the failing system test?

Reply:
#538 created.